Release Notes/511.md

```diff
@@ -30,6 +30,10 @@
   - Issue: https://github.com/apple/swift-syntax/issues/2267
   - Pull request: https://github.com/apple/swift-syntax/pull/2272
 
+- `EditorPlaceholderExprSyntax` & `EditorPlaceholderExprSyntax`:
+  - Description: `EditorPlaceholderExprSyntax` & `EditorPlaceholderExprSyntax` are now deprecated and not parsed anymore. They will now be parsed as identifiers.
```

Comment:
`Expr` twice, presumably wanted `Decl` as well? Super nitpick, but I'd also just use `and` rather than `&`.

```suggestion
- `EditorPlaceholderDeclSyntax` and `EditorPlaceholderExprSyntax` deprecated:
  - Description: `EditorPlaceholderExprSyntax` and `EditorPlaceholderExprSyntax` are now deprecated and placeholders are instead parsed as identifiers within a `MissingDeclSyntax` or `DeclReferenceExprSyntax`.
```

Reply:
I tell you. This time I didn't want any copy/paste error. But it happened again.. 🤦‍♂️

but it's fixed and nothing is to nitpick